code/modules/research/xenobiology/crossbreeding/charged.dm

```diff
@@ -205,7 +205,7 @@ Charged extracts:
 	addtimer(CALLBACK(src, .proc/boom), 50)
 
 /obj/item/slimecross/charged/oil/proc/boom()
-	explosion(get_turf(src), 3, 2, 1) //Much smaller effect than normal oils, but devastatingly strong where it does hit.
+	explosion(get_turf(src), 2, 3, 4) //Much smaller effect than normal oils, but devastatingly strong where it does hit.
```

Comment:
It specifically says "devastatingly strong where it does hit". This makes me think all hit tiles being dev tiles is intended.

Reply:
I'm having a hard time believing that a 3, 2, 1 explosion is intended. Would be a different story if it were 3, 0, 0.

Reply:
it's intended but it's also stupid, and no explosion should have major and minor smaller than devastation (except for 0) because that's just pointless.